aries-site/src/examples/templates/hub-spoke-navigation/HubSpokeCardsExample.js

```diff
@@ -0,0 +1,322 @@
+  Server,
+  Servers,
+  ShieldSecurity,
+  UnorderedList,
+} from 'grommet-icons';
+
+// Mock application page data
+const pages = [
+  {
+    name: 'Home',
+    layout: 'cards',
+    children: ['Devices', 'Performance', 'Users', 'Remote Access', 'Security'],
+  },
+  {
+    name: 'Performance',
+    icon: <Optimize />,
+    layout: 'cards',
+    parent: 'Home',
+    children: ['Sessions', 'Logs', 'Diagnostics'],
+  },
+  {
+    name: 'Devices',
+    icon: <Servers />,
+    layout: 'cards',
+    parent: 'Home',
+    children: ['Server Cluster', 'Beacon Cluster'],
+  },
+  {
+    name: 'Users',
+    icon: <Group />,
+    layout: 'list',
+    parent: 'Home',
+    children: new Array(17).fill(''),
+  },
+  {
+    name: 'Sessions',
+    icon: <ListIcon />,
+    layout: 'list',
+    parent: 'Performance',
+    children: new Array(17).fill(''),
+  },
+  {
+    name: 'Logs',
+    icon: <UnorderedList />,
+    layout: 'list',
+    parent: 'Performance',
+    children: new Array(17).fill(''),
+  },
+  {
+    name: 'Remote Access',
+    icon: <Location />,
+    layout: 'panes',
+    parent: 'Home',
+  },
+  {
+    name: 'Security',
+    icon: <ShieldSecurity />,
+    layout: 'panes',
+    parent: 'Home',
+  },
+  {
+    name: 'Diagnostics',
+    icon: <Overview />,
+    layout: 'panes',
+    parent: 'Performance',
+  },
+  {
+    name: 'Server Cluster',
+    icon: <Cluster />,
+    layout: 'cards',
+    parent: 'Devices',
+    children: ['Server A', 'Server B'],
+  },
+  {
+    name: 'Beacon Cluster',
+    icon: <Cluster />,
+    layout: 'cards',
+    parent: 'Devices',
+    children: ['Beacon A', 'Beacon B', 'Beacon C', 'Beacon D'],
+  },
+  {
+    name: 'Server A',
+    icon: <Server />,
+    layout: 'panes',
+    parent: 'Server Cluster',
+  },
+  {
+    name: 'Server B',
+    icon: <Server />,
+    layout: 'panes',
+    parent: 'Server Cluster',
+  },
+  {
+    name: 'Beacon A',
+    icon: <LocationPin />,
+    layout: 'panes',
+    parent: 'Beacon Cluster',
+  },
+  {
+    name: 'Beacon B',
+    icon: <LocationPin />,
+    layout: 'panes',
+    parent: 'Beacon Cluster',
+  },
+  {
+    name: 'Beacon C',
+    icon: <LocationPin />,
+    layout: 'panes',
+    parent: 'Beacon Cluster',
+  },
+  {
+    name: 'Beacon D',
+    icon: <LocationPin />,
+    layout: 'panes',
+    parent: 'Beacon Cluster',
+  },
+];
+
+const getPageDetails = pageName => pages.find(page => page.name === pageName);
+
+export const HubSpokeCardsExample = ({ mobile }) => {
+  const [currentPage, setCurrentPage] = React.useState(getPageDetails('Home'));
+  const [subPages, setSubPages] = React.useState();
+
+  React.useEffect(() => {
+    const pageDetails =
+      currentPage.children &&
+      currentPage.children.map(child =>
+        pages.find(page => page.name === child),
+      );
+    setSubPages(pageDetails);
+  }, [currentPage]);
+
+  const { children, layout, name, parent } = currentPage;
+
+  return (
+    <ResponsiveContext.Provider value={mobile && 'small'}>
```

Comment:
one thing to note is that once my example PR is merged I moved this `  <ResponsiveContext.Provider value={mobile && 'small'}>` to be within `example.js` file so we wont need this here

Reply:
Also not seeing a big change for mobile maybe Im missing something ?

Reply:
I messed that up when merging master into this branch. Sorry!

Reply:
> one thing to note is that once my example PR is merged I moved this ` <ResponsiveContext.Provider value={mobile && 'small'}>` to be within `example.js` file so we wont need this here

Yay!